phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -3063,60 +3063,62 @@ public boolean isViewReferenced() {
              */
             EncodedCQCounter cqCounterToBe = tableType == PTableType.VIEW ? NULL_COUNTER : cqCounter;
             PTable table = new PTableImpl.Builder()
-                    .setType(tableType)
```

Comment:
Here we are adding just 1 filed to builder method. Can we please undo the formatting changes which is making the diff look more than actual changes.

Reply:
Sorry for the inconvenience, but the formatting changes fix the spacing from being incorrect to correct. GitHub does have a "Hide whitespace changes" in the setting icon near the top of the page, btw, to make it easier to ignore.

Reply:
I know this is a very nit pick.. but for split lines the spacing rule is 8 spaces. For conditional/loop statements body I agree it is 4 spaces.